distributed-executor/scheduled-executor-retrieve-all-tasks/src/main/java/BasicTask.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.io.Serializable;
+import java.util.concurrent.Callable;
+
+public class BasicTask implements Callable, Serializable {
```

Comment:
Name suggestion: `EchoTask` (since it just returns its value)

Would be the same for all other `BasicTask` classes.